GDJS/Runtime/runtimescene.js

```diff
@@ -521,6 +521,14 @@ gdjs.RuntimeScene.prototype.updateObjectsForces = function() {
  * @param obj The object to be added.
  */
 gdjs.RuntimeScene.prototype.addObject = function(obj) {
+    if ( !this._objects.containsKey(obj.name) ) {
```

Comment:
If the object is not registered in scene register it using the same calls as in loadObjects